src/main/java/nl/opengeogroep/safetymaps/server/stripes/SafetyConnectProxyActionBean.java

```diff
@@ -83,121 +87,137 @@ public Resolution proxy() throws Exception {
+            final String responseContent = client.execute(req, new ResponseHandler<String>() {
+                @Override
+                public String handleResponse(HttpResponse hr) {
+                    log.debug("proxy for user " + context.getRequest().getRemoteUser() + " URL " + req.getURI() + ", response: " + hr.getStatusLine().getStatusCode() + " " + hr.getStatusLine().getReasonPhrase());
+                    contentType.setValue(hr.getEntity().getContentType().getValue());
+                    try {
+                        return IOUtils.toString(hr.getEntity().getContent(), "UTF-8");
+                    } catch(IOException e) {
+                        log.error("Exception reading HTTP content", e);
+                        return "Exception " + e.getClass() + ": " + e.getMessage();
                     }
-                });
-                
-                final String authorizedContent;
-                if (path.toLowerCase().startsWith("incident") && qs != null && qs.length() > 0) {
-                    authorizedContent = applyAuthorizationToIncidentContent(content, c);
-                } else if (path.toLowerCase().startsWith("incident") && (qs == null || qs.length() == 0)) {
-                    throw new Exception("Incident request incomplete.");
-                } else if(path.toLowerCase().startsWith("eenheidlocatie")) {
-                    authorizedContent = applyAuthorizationToEenheidLocatieContent(content);
-                } else {
-                    authorizedContent = content;
                 }
-
-                return new Resolution() {
-                    @Override
-                    public void execute(HttpServletRequest request, HttpServletResponse response) throws Exception {
-                        String encoding = "UTF-8";
-
-                        response.setCharacterEncoding(encoding);
-                        response.setContentType(contentType.getValue());
-
-                        OutputStream out;
-                        String acceptEncoding = request.getHeader("Accept-Encoding");
-                        if(acceptEncoding != null && acceptEncoding.contains("gzip")) {
-                            response.setHeader("Content-Encoding", "gzip");
-                            out = new GZIPOutputStream(response.getOutputStream(), true);
-                        } else {
-                            out = response.getOutputStream();
-                        }
-                        IOUtils.copy(new StringReader(authorizedContent), out, encoding);
-                        out.flush();
-                        out.close();
-                    }
-                };
-            } catch(IOException e) {
-                log.error("Failed to write output:", e);
-                return null;
+            });
+            
+            final String content;
+            // Data that could lead to an person needs to be authorized or filtered. 
```

Comment:
I'd comment `// Filter the response from the webservice to remove any data that the user is not authorized for.`